modeller/chapter.md

```diff
@@ -869,18 +897,19 @@ One major simplification in this project is in the user interface. A production
 complete user interface, which would necessitate a much more sophisticated events system instead of our simple callback system.
 
 We could do further experimentation to add new features to this project. Try one of these:
-* Add a Node type to support triangle meshes for arbitrary shapes.
-* Add an Undo stack, to allow undo/redo of modeller actions.
-* Save/Load the design using a 3D file format like [DXF](http://en.wikipedia.org/wiki/AutoCAD_DXFhttp://en.wikipedia.org/wiki/AutoCAD_DXF).
+
+* Add a `Node` type to support triangle meshes for arbitrary shapes.
+* Add an undo stack, to allow undo/redo of modeller actions.
+* Save/load the design using a 3D file format like DXF.
```

Comment:
Removed link to Wikipedia as per our [editing style](https://github.com/aosabook/500lines/wiki/Editing-Style-Guide).